contrib/debian/README.md

```diff
@@ -15,7 +15,7 @@ To install:
 
 If you build yourself, you will either need to modify the paths in
 the .desktop file or copy or symlink your dash-qt binary to `/usr/bin`
-and the `../../share/pixmaps/dash128.png` to `/usr/share/pixmaps`
+and the `../../share/pixmaps/bitcoin128.png` to `/usr/share/pixmaps`
```

Comment:
Actually, if we want to avoid conflicts with bitcoin packages imo it's time to fix bitcoin->dash for pixmaps https://github.com/dashpay/dash/tree/master/share/pixmaps (and references to them)